scraper/src/strategies/default_strategy.py

```diff
@@ -104,6 +104,9 @@ def get_records_from_dom(self, current_page_url=None):
             # We only save content for the 'text' matches
             content = None if current_level != 'content' else self.get_text(node, self.get_strip_chars(current_level, selectors))
 
+            if (content is None or content == "") and current_level == 'content':
```

Comment:
You could even use bool(content)
https://infohost.nmt.edu/tcc/help/pubs/python/web/bool-function.html